skyfield/functions.py

```diff
@@ -19,6 +19,16 @@ def dots(v, u):
     """
     return (v * u).sum(axis=0)
 
+def crosses(v, u):
```

Comment:
Ah, so cross products now come to the fore! Could you try testing the code with `_cross()` from `sgp4lib.py`? I have been wanting to try it on another use-case but hadn't found a significant one yet. This could be a great chance to see if it speeds up or slows down (or breaks?) another computation besides the one use case in that library.

Reply:
When I swapped that in it worked perfectly!